ballerina/tests/batch-execute-query-test.bal

```diff
@@ -96,6 +99,30 @@ function batchInsertIntoDataTableFailure3() {
     test:assertTrue(result is ApplicationError);
 }
 
+@test:Config {
+    groups: ["batch-execute"]
+}
+function batchInsertNegative() returns error? {
+    string path = "../ballerina/tests/resources/sample/parameterized_query.bal";
+    Process process = check exec("ballerina", {}, (), "version");
```

Comment:
Ballerina is not installed in the machine. You need to call the ballerina.bat script from the target folder.

Reply:
Removed this in 2406fa6